core-services/release-controller/README.md

```diff
@@ -53,10 +53,14 @@ To abort/remove a release tag:
 
 * `deploy-*.yaml` - the controller definitions
 * `images-*.yaml` - definitions of the image streams used by the publish / build rules
-* `releases/release-*.json` - definitions of the build and publishing rules for sets of images
-* `repos/ocp-*-*.repo` - definitions where RPMs are loaded from for OCP builds
+* `_releases/release-*.json` - definitions of the build and publishing rules for sets of images
+* `_repos/ocp-*-*.repo` - definitions where RPMs are loaded from for OCP builds
 * `rpms-*.yaml` - definitions of the services that exposed mirrored RPMs within the clutser
 
+### Release informers
+
+Release informers are configured via the `verify` property in `_releases/release-*.json`. To update TestGrid after making changes, use [`testgrid-config-generator`][testgrid-config-generator].
```

Comment:
```suggestion
Inclusion in testgrid dashboards is driven in part via the `verify` property in `_releases/release-*.json`. To update TestGrid after making changes, use [`testgrid-config-generator`][testgrid-config-generator].  For more details on how the dashboards are generated based on the various configuration inputs, see the testgrid-config-generator docs.
```

Reply:
I was aiming to cover both release-controller configuration and the interaction with Test Grid.  Let's roll this discussion into [the earlier thread][1] until we sort our whether that scoping attempt was too broad or not.

[1]: https://github.com/openshift/release/pull/7793/#discussion_r405140918